db/migrate/20240111112201_add_post_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostRefToLikes < ActiveRecord::Migration[7.1]
+  def change
+    add_reference :likes, :author, foreign_key: { to_table: 'users' }
+  end
```

Comment:
- [ ] As per the ERD diagram, The foreign key between likes and posts tables should be `post_id`, not `author_id` as you did here. Please fix that.

![image](https://github.com/yusufyusie/blog_app/assets/68796232/d5d0908f-4495-4adc-855d-94e4a3293be7)
```suggestion
  def change
    add_reference :likes, :post, null: false, foreign_key: true
  end
```